app/views/search/index.html.erb

```diff
@@ -187,6 +187,29 @@
             <%= t'Samples' %>
           </a>
         </li>
+
+        <% @repository_search_count.each do |team, results| %>
+          <li class="repositories-team <%= 'active' if results[:count] > 0 %>">
+            <span class="fa fa-cubes"></span>
+            <%= t('Repositories_team', team: team) %>
+          </li>
+          <% results[:repositories].each do |repository, values| %>
+            <li role="presentation"
+                class="
+                <%= "active" if @search_category.present? and @search_category == :repositories and @repository.id == values[:id] %>
```

Comment:
Ì would omit `@search_category.present?` since it would work without it. Your call.